### PR TITLE
Add cases for bug scenarios

### PR DESCRIPTION
**Pull request by sunchengzhu**
1. adjust native transfer case
2. add bn256 case

### CODE REVIEW

**Comment by sunchengzhu** [comment]
This pr needs to be merged after v1.13.0-rc1 is released,because `skip` in `it.skip` need to be removed.

File: contracts/test/bn256.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        // console.log(`before tx from_balance(${from.substring(0, 6)}):${from_balance}`)
+        const tx = await ethers.provider.send("eth_sendTransaction", [{
+            from,
+            to,
+            "gas": gas,
+            "gasPrice": gasPrice,
+            "data": data
+        }])
+        const receipt = await getTxReceipt(ethers.provider, tx, 100)
+        expect(receipt.status).to.be.equal(0);
```
Comment (Flouse) [inline contracts/test/bn256.js:39]:
```suggestion
        // receipt.status: 1 (success) or 0 (failure)
        expect(receipt.status).to.be.equal(0);
```

File: contracts/test/bn256.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        await contract.deployed();
+    });
+
+    it("0x06 legal input", async () => {
+        const tx = await contract.callBn256Add(numberToBytes32(1), numberToBytes32(2), numberToBytes32(1), numberToBytes32(2));
+        const receipt = await tx.wait();
+        expect(receipt.status).to.be.equal(1);
+    })
+
+    it.skip("0x06 illegal input", async () => {
```
Comment (Flouse) [inline contracts/test/bn256.js:21]:
```suggestion
    // TODO: remove `skip` after deploying Godwoken v1.13+.
    it.skip("0x06 illegal input", async () => {
```

File: contracts/test/nativeTransfer.mjs [hunk 3]
```diff
@@ -115,7 +100,32 @@ describe("transfer failed", function () {
...
             const from_balance_sent = await ethers.provider.getBalance(from)
             const to_balance_sent = await ethers.provider.getBalance(to)
-            console.log(`after transfer from balance:${from_balance_sent} to balance:${to_balance_sent}`)
+            console.log(`after transfer from_balance(${from.substring(0, 6)}):${from_balance_sent} to_balance(${to.substring(0, 6)}):${to_balance_sent}`)
+            expect(from_balance).to.be.equal(from_balance_sent)
+            expect(to_balance).to.be.equal(to_balance_sent)
+        }
+    }).timeout(15000)
+
+    it.skip("gasLimit not enough but enough for 21000", async () => {
```
Comment (Flouse) [inline contracts/test/nativeTransfer.mjs:110]:
```suggestion
    // TODO: remove `skip` after deploying Godwoken v1.13+.
    it.skip("gasLimit not enough but enough for 21000", async () => {
```

**Review by Flouse — APPROVED** [review]

**Pull request merged**